### PR TITLE
Questions in SubCommand_Create.java

### PR DESCRIPTION
**Pull request by Chris6ix**
1.What exactly is meant by "3rd-plugin-build-check-failed"? (line 130)
2.What is this "Util" (line 147)

### CODE REVIEW

File: src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Create.java [hunk 1]
```diff
@@ -127,7 +127,7 @@ public void onCommand(
             if (p.isOnline()) {
                 Result result = plugin.getPermissionChecker().canBuild(p, b);
                 if (!result.isSuccess()) {
-                    MsgUtil.sendMessage(p, MsgUtil.getMessage("3rd-plugin-build-check-failed", p, result.getMessage()));
+                    MsgUtil.sendMessage(p, MsgUtil.getMessage("->3rd-plugin-build-check-failed", p, result.getMessage()));
```
Comment (Ghost-chu) [inline src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Create.java:130]: This is a string key and points to a language file string, you shouldn't touch it.

File: src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Create.java [hunk 2]
```diff
@@ -144,7 +144,7 @@ public void onCommand(
                 // As of the new checking system, most plugins will tell the
                 // player why they can't create a shop there.
                 // So telling them a message would cause spam etc.
-                Util.debugLog("Util report you can't build shop there.");
+                Util.debugLog("->Util report you can't build shop there.");
```
Comment (Ghost-chu) [inline src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Create.java:147]: Ignore it, it's debug log

**Review by Ghost-chu — CHANGES_REQUESTED** [review]
There something that needs to pay attention to.

**Comment by Chris6ix** [comment]
okay

**Pull request closed without merging**